### PR TITLE
Add security compliance logo

### PR DESCRIPTION
**Pull request by Lukeaber**
### Description of proposed changes
Adding the new security compliance logo so that we can use it in the new release of comply.  

### Screenshot of proposed changes

Screenshot 2024-03-13 at 11 24 33

Screenshot 2024-03-13 at 13 22 36

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## react-components 5.34.1 (2024-03-13)
```
Comment (mttsltrs) [inline CHANGELOG.md:1]: Should this be 5.34.2?

**Comment by sean-mckenna** [comment]
Your screenshot looks a bit different than the figma I looked at.

Screenshot 2024-03-13 at 14 34 18

**Comment by sean-mckenna** [comment]
Also just checking you followed this process https://www.figma.com/file/JKwXo5WqdvIuXUFcG0Fk0s/%5BPE-37533%2C-PE-37531%2C-PE-37532%2C-PE-37534%5D-PE-Suite-discoverability?type=design&node-id=742-53257&mode=design&t=xDKZDwk6soKFcW6J-4

**Comment by Lukeaber** [comment]
Yeah used svgo to strip down the svg and manually cut out svg element etc. as described on the webste.

**Review by sean-mckenna — APPROVED** [review]

**Pull request merged**